tornado_elasticsearch.py

```diff
@@ -343,13 +369,10 @@ def exists(self, index, id, doc_type='_all', params=None):
             performing the operation
         :arg routing: Specific routing value
         """
-        try:
-            self.transport.perform_request('HEAD',
+        result = yield self.transport.perform_request('HEAD',
```

Comment:
Correct. Due to the request being HEAD it will just return true or false. I don't think it makes sense to use an exception to handle a case that is relatively normal/expected behavior.